src/components/ModalState/styled.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import { em } from 'polished';
+
+export const variants = ['success', 'warning', 'danger'] as const;
+export type Variant = typeof variants[number];
+
+export const ModalStateTitle = styled.div`
+  font-size: ${({ theme }) => em(theme.honeycomb.size.normal)};
+  text-align: center;
+  color: ${({ theme }) => theme.honeycomb.color.text.normal};
+  margin-top: ${({ theme }) => em(theme.honeycomb.size.normal)};
+`;
+
+export const ModalStateContent = styled.div`
+  color: ${({ theme }) => theme.honeycomb.color.text.grey};
+  font-size: ${({ theme }) => em(theme.honeycomb.size.reduced)};
+  text-align: center;
+  margin-top: ${({ theme }) => em(theme.honeycomb.size.normal / 2)};
```

Comment:
```suggestion
  margin-top: ${({ theme }) => em(theme.honeycomb.size.small, theme.honeycomb.size.reduced)};
```

Reply:
Use `theme.honeycomb.radius.normal` here instead since marginTop is 8px

Reply:
`8px` does not exist in our current theme/design system, so just `small` is good enough.